### DISCUSSION TITLE
Demo or test

### DISCUSSION THREAD

**Discussion by rabol** [lucky-media/landtamic#12] [Q&A]
Hi

is there any demo or test version available ?

**Comment by lokmanm** [comment]
> Hi
> 
> is there any demo or test version available ?

Hi @rabol we have a working site [here](https://landtamic.com/) and also a video explaining how it works [here](https://www.youtube.com/watch?v=XBa6MWjCZCs).

As for the live demo that's something that we don't have yet.

Let me know if you have any questions.

**Reply by rabol** [reply]
thanks!

I don't see any 'drop-down' menu, is that not supported ?

In CP on can 'move' items under a parent
